Macros/e3 Includes/e3_Basics.inc

```diff
@@ -1620,6 +1620,9 @@ SUB basics_MacroSettings
 /RETURN
 
 SUB basics_CharacterSettings
+/if ((${Select[${Me.Class.ShortName},CLR,DRU,ENC,MAG,NEC,SHM,WIZ]})) /call WriteToIni "${Character_Ini},Misc,End MedBreak in Combat(On/Off)" Off else /if (${Me.Class.ShortName.NotEqual[BRD]}) /call WriteToIni "${Character_Ini},Misc,End MedBreak in Combat(On/Off)" On
```

Comment:
Could you make else /if multi-line and use } { to separate the scopes? It's hard to read a single line with multiple conditions, and frankly, I don't trust mq2 to parse it correctly. I mean, clearly you've tested it and it's working for you, but I think it would be better to make it clear with multi-lines and defined scopes.

Reply:
Done!